spec/unit/action/runner_spec.rb

```diff
@@ -170,6 +171,37 @@ def call
     end
   end
 
+  describe "configuring git credentials" do
+    it 'errors if both token and key paths are passed' do
+      runner = described_class.new({ 'token-path': '/nonexistent',
+                                     'sshkey-path': '/also/fake' }, %w[args yes], action_class)
+      expect{ runner.call }.to raise_error(R10K::Error, /Cannot specify both/)
+    end
+
+    it 'passes sshkey path to deployment' do
```

Comment:
(nit) should this say something like 'saves the sshkey path in settings hash' or something? since I don't think we're passing to deployment anymore?